api/chaos/experimentation/v1/experimentation.proto

```diff
@@ -31,6 +33,14 @@ message GetExperimentsResponse {
   repeated Experiment experiments = 3;
 }
 
+message GetExperimentRunConfigPairDetailsRequest {
```

Comment:
this is a pretty long name. maybe GetExperimentRunRequest? or something similar?